_docs/tasks/traffic-management/circuit-breaking.md

```diff
@@ -19,7 +19,7 @@ This task demonstrates the circuit-breaking capability for resilient application
   which will be used as the backend service for our task
   
   ```bash
-  kubectl apply -f <(istioctl kube-inject -f samples/httpbin/httpbin.yaml)
+  kubectl apply -f <(istioctl kube-inject --debug -f samples/httpbin/httpbin.yaml)
```

Comment:
What does this fix? Are debug images and/or additional logging required for this task?